app/assets/javascripts/geoblacklight/modules/metadata.js

```diff
@@ -0,0 +1,16 @@
+
+var setRefUrl = function setRefUrl() {
```

Comment:
Could this be implemented in a way where a global variable function isn't added? I'm concerned with adding to the global scope here.